src/Packagist/WebBundle/Entity/PackageRepository.php

```diff
@@ -279,7 +279,7 @@ public function getFilteredQueryBuilder(array $filters = array())
             $qb->leftJoin('v.tags', 't');
         }
 
-        $qb->orderBy('p.id', 'DESC');
```

Comment:
@stof I replaced this order here: https://github.com/composer/packagist/pull/666/files#diff-fa57170b5eba2155e6e39953836f12efR61

`getFilteredQueryBuilder` method is used by `UserController` and `PackageController` but not with the same order. I had to do this way.

The order is not changed at all in any case, except for abandoned packages.